src/Resources/config/routing.yml

```diff
@@ -2,10 +2,10 @@ monitor.info:
     path: /info
     methods: [GET]
     defaults:
-        _controller: openconext.monitor.controller.info:infoAction
+        _controller: openconext.monitor.controller.info
```

Comment:
I'm not the biggest fan of using the __invoke method over a named controller action. But alas, this works too!

Reply:
I run into some trouble with removing the deprecation with the explicit method names. That was the easiest way that works. So i choose that solution